testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/RemoveManagementRealmTestCase.java

```diff
@@ -52,6 +53,7 @@
  */
 @RunWith(WildflyTestRunner.class)
 @ServerControl(manual = true)
+@Ignore("Random failures due to aesh/CLI close sequence, Unignore it when CLI relies on aesh-readline.")
```

Comment:
Can you reference a JIRA in this comment? If you plan to just reschedule WFCORE-2679 to 4.0 and not treat this as a resolution, then it can be that. That's simple and is the usual pattern. Otherwise it should be something else, perhaps the aesh-readline JIRA or some other thing.